dsl_parser/elements/imports.py

```diff
@@ -169,6 +170,42 @@ def _get_resource_location(resource_name,
     return None
 
 
+def _extract_import_parts(import_url,
+                          resources_base_path,
+                          current_resource_context=None):
+    """
+    :param import_url: a string which is the import path
+    :param resources_base_path: In case of a relative file path, this
+                                is the base path.
+    :param current_resource_context: Current import statement,
+    :return: Will return a breakdown down of the URL to namespace, import_url.
+            If the import is not namespaced, the returned namespace will be
+            None.
+    """
+    namespace_op_location = utils.find_value_namespace(import_url)
```

Comment:
The split option is problematic because there could be multi namespace enabled and I am looking for the last one (the namespace from the start of the string)

Reply:
Imporve the code a little